andela/selene-ah-backend
# #162756461 Merge user and profile table

#### What does this PR do?
Merge user and profile table
#### Description of Task to be completed?
- Move all fields into the user table
- Remove the profile table
- Remove all the tests, references to the profile model

#### How should this be manually tested?
git clone this branch
Run `npm run db:reset` and `npm run db:migrate`
Check your db to verify if a `Profile` model exist.

#### Any background context you want to provide?
##### PS - You would need to manually delete your existing db to avoid any errors.

#### What are the relevant pivotal tracker stories?
[#162756461](https://www.pivotaltracker.com/story/show/162756461)

#### Screenshots (if appropriate)
Nil
#### Questions:
Nil
